config/dependency-lint.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = {
+
```

Comment:
Since we run dependency-lint through a cli-command, we can disable the test generation: https://github.com/salsify/ember-cli-dependency-lint#lint-tests At the moment the linting is being done twice.